apps/web/app/environments/[environmentId]/(actionsAndAttributes)/actions/ActionClassesTable.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import { Button } from "@formbricks/ui";
+import { CursorArrowRaysIcon } from "@heroicons/react/24/solid";
+import { useState } from "react";
+import AddNoCodeActionModal from "./AddNoCodeActionModal";
+import ActionDetailModal from "./ActionDetailModal";
+import { useRouter } from "next/navigation";
+import { TAction } from "@formbricks/types/v1/actions";
+
+export default function ActionClassesTable({
+  environmentId,
+  actionClasses,
+  children: [TableHeading, actionRows],
+}: {
+  environmentId: string;
+  actionClasses: TAction[];
+  children: [JSX.Element, JSX.Element[]];
+}) {
+  const router = useRouter();
+  const [isActionDetailModalOpen, setActionDetailModalOpen] = useState(false);
+  const [isAddActionModalOpen, setAddActionModalOpen] = useState(false);
+
+  const [activeActionClass, setActiveActionClass] = useState("" as any);
```

Comment:
why `any`? isn't it a string because you initialize it with `""`? or a complex object? please name a more accurate type

Reply:
oops, my mistake! handled!